acceptance/tests/vault/vault_namespaces_test.go

```diff
@@ -24,6 +25,13 @@ func TestVault_VaultNamespace(t *testing.T) {
 	cfg := suite.Config()
 	ctx := suite.Environment().DefaultContext(t)
 	ns := ctx.KubectlOptions(t).Namespace
+
+	ver, err := version.NewVersion("1.12.0")
```

Comment:
Does this mean that your other PR will change and you will pass in consul-version to the circle CI tests to match?

Reply:
Correct, when we add the new circleci jobs they will each now pass `-consul-image=<hashicorpdevpreview>/consul:1.11-dev` and `-consul-version=1.11-dev` and those will be both hard coded and not have to change. (and 1.12-* respectively)